docs/usecases/HCP_dataset.rst

```diff
@@ -0,0 +1,580 @@
+    │       │   ├── 05
+    │       │   │   └── 3d25959223e8173497fa7f747442b72c31671c
+    │       │   ├── 0b
+    │       │   │   └── 8d0edbf8b042998dfeb185fa2236d25dd80cf9
+    │       │   ├── 0d
+    │       │   │   └── a63a92497bd0996aa9398fc9e05783c5d0fe65
+    │       │   ├── [...]
+    │       │   │   └── [...]
+    │       │   ├── info
+    │       │   └── pack
+    │       ├── refs
+    │       │   ├── heads
+    │       │   │   ├── git-annex
+    │       │   │   └── master
+    │       │   └── tags
+    │       ├── ria-layout-version
+    │       └── ria-remote-ebce196a-b057-4c96-81dc-7656ea876234
+    │           └── transfer
+
+
+The :command:`datalad clone` command can understand this layout and install
+datasets from a RIA store based on their ID.
+
+.. findoutmore:: How would a datalad clone from a RIA store look like?
+
+   In order to get a dataset from a RIA store, :command:`datalad clone` needs
+   a RIA URL. It is build from the following components:
+
+   - a ``ria+`` identifier
+   - a path/url to the store in question. For store.datalad.org, this is
+     ``http://store.datalad.org``, but it could also be an SSH url, such as
+     ``ssh://juseless.inm7.de/data/group/psyinf/dataset_store``
+   - a pound sign (``#``)
+   - the dataset ID
+   - and optionally a version or branch specification (appended with a leading ``@``)
+
+   Here is how a valid :command:`datalad clone` command from the data store
+   for one dataset would look like:
+
+   .. code-block:: bash
+
+      datalad clone 'ria+http://store.datalad.org#d1ca308e-3d17-11ea-bf3b-f0d5bf7b5561' subj-01
+
+   But worry not! To get the HCP data, no-one will ever need to compose
+   :command:`clone` commands to RIA stores apart from DataLad itself.
+
+While its layout looks confusing, a RIA store is nothing but a clever storage
+solution, and users never consciously interact with the store to get the HCP
+datasets. On the lowest level, `store.datalad.org <http://store.datalad.org/>`__
+is a directory on a publicly accessible server that holds a great number of such
+bare repositories. The only important aspect of it for this usecase is that
+instead of by their names (e.g., ``100206``), datasets are stored and identified
+via their :term:`dataset ID` (the first two highlighted lines). The hidden section
+above demonstrated how this dataset ID becomes relevant to obtain the dataset.
+Importantly, after cloning a dataset from a RIA store, the dataset clone will
+have its usual file hierarchy and file names.
+
+A RIA store is used, because -- among other advantages -- its layout makes the
+store flexible and scalable. With datasets of sizes like the HCP project,
+especially scalability becomes an important factor. If you are interested in
+finding out why, you can find more technical details on RIA stores, their advantages,
+and even how to create and use one yourself in the use case :ref:`usecase_datastore`.
+
+.. todo::
+
+   Extend the datastore use case
+
```

Comment:
I would also cut this completely and ref the RIA use case further down

Reply:
Just to clarify: How much would you cut, the entire "Using a Remote Indexed Archive Store for dataset hosting" subsection?

Reply:
Just to clarify: How much would you cut, the entire "Using a Remote Indexed Archive Store for dataset hosting" section?